src/rules/converters.ts

```diff
@@ -257,10 +259,6 @@ export const converters = new Map([
     // ["ban", convertBan], // no-restricted-properties
     // ["import-blacklist", convertImportBlacklist], // no-restricted-imports
     // ["no-duplicate-variable", convertNoDuplicateVariable], // no-redeclare
-    // ["no-unused-expression", convertNoUnusedExpression], // no-unused-expressions
-    // ["no-void-expression", convertNoVoidExpression], // (no exact equivalent)
-    // ["quotemark", convertQuotemark], // quotes
-    // ["triple-equals", convertTripleEquals], // eqeqeq
```

Comment:
Can you undo these unrelated changes? If these comments should be removed feel free to send a separate PR.

Reply:
Sure! This commit is a merge conflit solve. I'll undo this.